content/QRadar-SIEM/102.es.md

```diff
@@ -0,0 +1,1079 @@
+
+> También habrá cuadros azules de "notas" como éste que contienen puntos de discusión adicionales de la narración que usted utilizaría en la conversación con el cliente mientras realiza una demostración basada en este contenido de laboratorio. Para el laboratorio, puede hojear rápidamente estas notas para obtener contexto adicional.
+
+Debe haber completado todos los pasos del [laboratorio de configuración](./qradar-siem/101.md). Esto incluye la ejecución de los scripts `noiseOn.sh` y `runUC.sh` para preparar la instancia de QRadar SIEM con los datos utilizados en los ejemplos.
+
+## Caso práctico de demostración
+
+> La regulación se ha vuelto crucial para la ciberseguridad a raíz de numerosas violaciones de datos de gran repercusión. Muchas normativas sectoriales y geográficas ejercen una presión significativa sobre las organizaciones para que cumplan los requisitos de protección de los datos de los clientes. Cada conjunto de normativas (HIPAA, PCI, GDPR, SOX, etc.) contiene diferentes definiciones y requisitos, todos los cuales afectan a la forma de trabajar de una organización. IBM Security QRadar proporciona configuraciones enriquecidas que pueden automatizar la recopilación, correlación y generación de informes de datos de cumplimiento normativo.
+>
+> El objetivo de este caso de uso es demostrar cómo la cartera SIEM de QRadar permite a los analistas de seguridad identificar, detectar y responder al comportamiento sospechoso de la red en relación con los activos protegidos por el cumplimiento.
+>
+> La protección de datos críticos y regulados es el núcleo de nuestra cartera de seguridad. Se trata de una tarea polifacética que requiere un enfoque multicapa.
+>
+> El propio QRadar ha obtenido certificaciones de cumplimiento vitales que validan la seguridad del producto para que pueda confiar en sus soluciones, incluyendo [\[DHS CDM Approved Product List\]](https://urldefense.proofpoint.com/v2/url?u=https-3A__www.gsa.gov_portal_content_177883&d=DwMGaQ&c=jf_iaSHvJObTbx-siA1ZOg&r=QGEyT_hBYSpDAVjwcheX88fk8GoidzLRH7G2kTRmXXU&m=BmRMr9b_DO4jRx5UvQQiv2zGc_tzvS3WGjKtUdf1dQY&s=n6mOtr-30I2GvDco_nfvVCCYUHQrUNPj4R6QD2wSAQA&e=), FIPS 140-2 Nivel 1 y Nivel 2: Cert [\[#2737\]](https://urldefense.proofpoint.com/v2/url?u=https-3A__csrc.nist.gov_projects_cryptographic-2Dmodule-2Dvalidation-2Dprogram_certificate_2737&d=DwMGaQ&c=jf_iaSHvJObTbx-siA1ZOg&r=QGEyT_hBYSpDAVjwcheX88fk8GoidzLRH7G2kTRmXXU&m=LC928xze9wi-Sn74UcptQ6H7tFPDNnMRrNvIGs6UOko&s=Vy80fB6BHvVfI60Ie--oJutIDgs5P94nY_zdqQjJ27Q&e=) y[\[#2554](https://urldefense.proofpoint.com/v2/url?u=https-3A__csrc.nist.gov_projects_cryptographic-2Dmodule-2Dvalidation-2Dprogram_certificate_2554&d=DwMGaQ&c=jf_iaSHvJObTbx-siA1ZOg&r=QGEyT_hBYSpDAVjwcheX88fk8GoidzLRH7G2kTRmXXU&m=LC928xze9wi-Sn74UcptQ6H7tFPDNnMRrNvIGs6UOko&s=VRFORbKjOHXct0OXDPI74ookxYT1oHGoTq2NMoiqOjE&e=),] y habilitando [\[NIST 800-53 Security Controls\]](https://www.ibm.com/downloads/cas/ODQ9P07O).
```

Comment:
The `DHS CDM Approved Product List`  link points to a 404 page on the GSA page for both English & Spanish content

Reply:
Correct link for `DHS CDM Approved Product List` is https://www.cisa.gov/resources-tools/programs/continuous-diagnostics-and-mitigation-cdm-program/program-approved-products-list-apl - there is an excel downloadable file and I have verified that QRadar SIEM is present in the file. To avoid the url obfuscator the correct link markdown is:

```
> El propio QRadar ha obtenido certificaciones de cumplimiento vitales que validan la seguridad del producto para que pueda confiar en sus soluciones, incluyendo [[DHS CDM Approved Product List]](<<https://www.cisa.gov/resources-tools/programs/continuous-diagnostics-and-mitigation-cdm-program/program-approved-products-list-apl>), FIPS 140-2 Level 1 and Level 2: Cert [[#2737]](<https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/2737>) and [#2554](hhttps://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/2554), and enabling [[NIST 800-53 Security Controls]](<https://www.ibm.com/downloads/cas/ODQ9P07O>).
```